frontend/app/scripts/controllers/popups/waitListCtrl.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('frontendApp')
+  .controller('WaitListCtrl', ['participants','$scope', function(participants, $scope) {
```

Comment:
angular dependencies is first ($scope, $rootScoope, etc), then library e.g. ngTable, then our services. Please.